wire-swift-generator/src/main/java/com/squareup/wire/swift/SwiftGenerator.kt

```diff
@@ -158,6 +158,37 @@ class SwiftGenerator private constructor(
       else -> null
     }
 
+  private val Field.defaultedValue: CodeBlock?
+    get() = default?.let {
+      return defaultFieldInitializer(type!!, it)
+    } ?: when {
+      !isOptional -> null
```

Comment:
what about non optional fields? I still think we should emit those? I think I got a Swift compiler error...